package.json

```diff
@@ -5,23 +5,28 @@
   "main": "index.js",
   "scripts": {
     "build": "webpack --production",
-    "start": "webpack-dev-server --content-base dist/"
+    "start": "webpack-dev-server --content-base dist/",
+    "dev": "webpack-dev-server --content-base dist/"
   },
   "author": "",
   "devDependencies": {
-    "xb-common-lib": "git@bitbucket.org:sprinklr/xb-common-lib.git",
     "alt": "^0.18.4",
     "alt-utils": "^1.0.0",
     "babel-core": "^6.7.7",
     "babel-loader": "^6.2.4",
     "babel-preset-es2015": "^6.6.0",
     "babel-preset-react": "^6.5.0",
+    "babel-preset-stage-1": "^6.5.0",
+    "babel-relay-plugin": "^0.9.0",
     "i18n-webpack-plugin": "^0.2.7",
     "react": "^15.0.1",
```

Comment:
Nope, those are used only during development. This application is packaged by webpack